embed/embed_dev.go

```diff
@@ -16,16 +16,16 @@ var Content fs.FS
 func init() {
 	wd, err := os.Getwd()
 	if err != nil {
-		log.Panicf("cannot get current directory: %v", err)
+		log.Panicf("Cannot get current directory: %v", err)
 	}
 
 	embedDir := filepath.Join(wd, "embed")
 	stat, err := os.Stat(embedDir)
 	if err != nil {
-		log.Panicf("cannot stat embed directory: %v", err)
+		log.Panicf("Cannot stat embed directory: %v", err)
 	}
 	if !stat.IsDir() {
-		log.Panicf("embed directory is not a directory: %s", embedDir)
+		log.Panicf("Embed directory is not a directory: %s", embedDir)
```

Comment:
Internal errors in go generally prefer no capitalization. It's panic here tho so whatever :)

Reply:
Reverted this.